src/msg/message.cc

```diff
@@ -81,16 +72,52 @@ void CRMessageBase::Unsubscribe(const channel_id_t channel, CRNode* node) {
     }
 }
 
-cr_timestamp_nsec_t CRMessageBase::GetLatestDeliveredTime(const channel_id_t channel) {
+void SubscriptionMap::Dispatch(const CRMessageBasePtr& message) {
+    std::shared_lock lck(mtx_);
+    auto             subscription_find = map_.find(message->GetChannelId());
+    if (subscription_find == map_.end()) {
+        return;
+    }
+    auto& subscription_info = subscription_find->second;
+    for (auto& node : subscription_info.sub_list_) {
+        [[maybe_unused]] const bool success = node->AddMessageToRunner(message);
+        DCHECK(success);
```

Comment:
We don't have dbg build in CI, and we rarely have anyone doing that locally.

Suggest to avoid discrepancy between dbg/rel mode as much as possible.
For cheap checks, just run it in rel mode as well, won't really hurt.